src/components/SingleSourceRow.vue

```diff
@@ -65,6 +65,11 @@
   }
 
   .module-tile {
+    /* i know we have an scss transition but it didnt seem to actually work. so eventually this will be a part of that instead of manual probably */
```

Comment:
...and you answered the last question I asked from a different file here. Carry on!